src/tools/analyzer.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+permalink: /tools/analyzer
+title: "Static Analyzer"
+description: "The tool that analyzes and validates your Dart code."
+---
+
+The static analyzer evaluates your Dart code,
+checking for errors and warnings that are specified in the
+[Dart Language Specification](https://www.dartlang.org/docs/spec/).
+
+Many tools in the Dart ecosystem use the [analysis
+server](https://github.com/dart-lang/sdk/tree/master/pkg/analysis_server)
+to perform static analysis. For example:
+
+* Dart and Flutter plugins for IDEs (such as
+  [WebStorm](https://webdev.dartlang.org/tools/webstorm),
+  [IntelliJ](https://www.dartlang.org/tools/jetbrains-plugin), and
+  [Atom](https://atom.io/packages/dartlang))
+* [DartPad](https://www.dartlang.org/tools/dartpad)&mdash;browser-based
+  Dart playground
+* [Dart Dev Compiler (DDC)](https://github.com/dart-lang/sdk/tree/master/pkg/dev_compiler)&mdash;experimental development tool and transpiler for web apps
+* [`dartanalyzer`](https://github.com/dart-lang/sdk/tree/master/pkg/analyzer_cli#dartanalyzer
+)&mdash;command-line static analyzer for web or VM apps
+* [`flutter analyze`](https://flutter.io/debugging/#the-dart-analyzer)&mdash;command-line
+  static analyzer for mobile (Flutter) apps
+
+## Customizing static analysis
+
+You can customize the behavior of the analyzer or the linter
+by adding an analysis options file (`analysis_options.yaml`) to
+the package root of your project. The linter, an analyzer plugin,
```

Comment:
The fact that the lints are implemented as a plugin is an implementation detail, not something the user needs to know about. I'd prefer something like "The analysis options file allows you to enable the lint rules appropriate to your project."